addons/l10n_cz/data/tax_report.xml

```diff
@@ -6,7 +6,7 @@
         <field name="availability_condition">country</field>
         <field name="country_id" ref="base.cz"/>
         <field name="filter_hierarchy">optional</field>
-        <field name="filter_multi_company">disabled</field>
+        <field name="filter_multi_company">selector</field>
```

Comment:
Hmm, valid question. I asked Hesham, who recently added this tax report, but he got the specifications from a partner apparently that specified the field should be `disabled`. I wonder however if the partner took into account VAT Units ...

Reply:
@Olivier-olma Do you know if there's a valid reason to not have the tax report work with VAT Units in Czechia?

Reply:
@oco-odoo I checked with the partner that provided the initial details on the report, and he agrees we can change it to `tax_units`. Since it's not really used in Czechia apparently, it will not make a difference.